cluster/clean.sh

```diff
@@ -3,6 +3,11 @@
 source ./cluster/gocli.sh
 source ./hack/build/config.sh
 
+if [ "${KUBEVIRT_PROVIDER}" == "external" ]; then
+  echo "No clean up implemented for external providers!"
```

Comment:
Why no clean?

Reply:
cuz I'm a slacker :)  I ran into an issue trying to implement it:  https://github.com/kubevirt/containerized-data-importer/issues/774

I'll take another look and maybe do another iteration.

Reply:
Added clean functionality